contrib/rspamd/template.py

```diff
@@ -0,0 +1,62 @@
+pkgname = "rspamd"
+pkgver = "3.7.5"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCONFDIR=/etc/rspamd",
+    "-DENABLE_FASTTEXT=ON",
+    "-DENABLE_HYPERSCAN=OFF",
+    "-DENABLE_LUAJIT=OFF",
+    "-DENABLE_URI_INCLUDE=ON",
+    "-DRSPAMD_GROUP=_rspamd",
+    "-DRSPAMD_USER=_rspamd",
+    "-DSYSTEM_FMT=ON",
+    "-DSYSTEM_XXHASH=ON",
+    "-DSYSTEM_ZSTD=ON",
+    "-D_CAN_RUN=0",
+    "-DHAVE_ATOMIC_BUILTINS_EXITCODE=0",
```

Comment:
these 2 look weird, what are they for?

Reply:
Needed for cross compilation, cmake generates some .cmake files during cross compilation that need these